src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_validators.py

```diff
@@ -90,3 +90,16 @@ def validate_linux_host_name(namespace):
     if not found:
         raise CLIError('--name cannot exceed 63 characters and can only contain '
                        'letters, numbers, or dashes (-).')
+
+def validate_connector_name(namespace):
+    """Validates a string as a legal connector name.
+
+    This validation will also occur server-side in the kubernetes, but that may take
+    for a while. So it's more user-friendly to validate in the CLI pre-flight.
+    """
+    # https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md
+    regex = re.compile(r'^[a-z0-9]([a-z0-9\-]*[a-z0-9])?$')  # pylint:disable=line-too-long
```

Comment:
In this project's `.flake8` and `pylintrc` files we set max-line-length to 120. So it seems like this disable comment wouldn't be needed (and some lines below don't need to be wrapped). No need to change anything, I was just curious.

Reply:
Please remove the `pylint:disable=line-too-long` here.